test/integration/spec/localparticipant/publishunpublishtrack.js

```diff
@@ -238,6 +238,12 @@ describe('LocalParticipant: publishUnpublishTrack', function() {
         return;
       }
 
+      // eslint-disable-next-line no-warning-comments
+      // TODO(lrivas): Disabling DataTracks for Firefox P2P because more investigation is needed on the issue
```

Comment:
Do we have a separate jira ticket to track this todo?

Reply:
Yes, here is a ticket to follow up this later. https://twilio-engineering.atlassian.net/browse/VBLOCKS-4513